src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,23 @@
 package core.mate.academy.model;
 
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
-public class Bulldozer extends Machine {
+public class Bulldozer extends Machine implements MachineProducer<Bulldozer> {
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    @Override
+    public List<Bulldozer> get() {
+        return List.of(new Bulldozer(), new Bulldozer());
+    }
```

Comment:
i dont understand prev mentor's comment, Bulldozer model is not a producer, no need to implement from MachineProducer
instead, you should create separate package `producer`, and create Producers for each model, implemented MachineProducer<?> interface
also, according to the task, you should add some fields to this model
![image](https://github.com/mate-academy/jv-pecs/assets/73041091/e96a6970-9bac-40b2-864e-3d1b13d1e9be)

```suggestion
public class Bulldozer extends Machine {
    public Bulldozer() {
    }

    @Override
    public void doWork() {
        System.out.println("Bulldozer started to work");
    }
```